Example/Controllers/ContactsListSettingsTableViewController.swift

```diff
@@ -1,46 +1,96 @@
 import UIKit
 import MiniApp
 
+protocol ContactsListDelegate: class {
+    func contactsController(_ contactsController: ContactsListSettingsTableViewController, didSelect contact: [MAContact]?)
+}
+
 class ContactsListSettingsTableViewController: UITableViewController {
 
     var userContactList: [MAContact]? = []
-    internal static let contactName = "Example Name", contactEmail = "name@example.com"
+    weak var contactDelegate: ContactsListDelegate?
+    var selectedContacts = [MAContact]()
+    var allowMultipleSelection = true
```

Comment:
these new variables are here to support a contact picker mode in this controller (see main storyboard)